src/main/resources/swagger/agora.yaml

```diff
@@ -2077,27 +1904,13 @@ definitions:
       meta-version:
         type: string
         description: 'The version of this tool in the registry. Iterates when fields like the description, author, etc. are updated.'
-      contains:
-        description: An array of IDs for the applications that are stored inside this tool (for example `https://bio.tools/tool/mytum.de/SNAP2/1`)
-        type: array
-        items:
-           type: string
-      verified:
-        type: boolean
-        description: Reports whether this tool has been verified by a specific organization or individual
-      verified-source:
-        type: string
-        description: Source of metadata that can support a verified tool, such as an email or URL
-      signed:
-        type: boolean
-        description: Reports whether this tool has been signed.
```

Comment:
I don't think you should remove these properties. We do return these keys in the json responses; they're just constantly blank/false. Either leave them as-is defined by the spec, or feel free to add e.g. "always returns empty string" for our implementation.

Update: same issue for other models below. Bah, maybe this means you need to update orch swagger too; I didn't look closely enough over there!

Reply:
They are basically the same - I'm trying to make the same changes in both places.